lib/spl-build.js

```diff
@@ -451,65 +487,67 @@ export default {
   },
 
   handleBuildCallback(e) {
-    const selectedComp = this.mainCompositePickerView.mainComposite;
-    if (selectedComp) {
-      this.mainCompositeSelectorPanel.hide();
-      if (this.apiVersion === CONF_API_VERSION_V5) {
-        const fqn = this.namespace ? `${this.namespace}::${selectedComp}` : `${selectedComp}`;
-        const toolkitRootPath = atom.config.get(CONF_TOOLKITS_PATH);
-        let messageHandler = MessageHandlerRegistry.get(fqn);
-        if (!messageHandler) {
-          this.consoleService = this.consumeConsoleService({ id: fqn, name: fqn });
-          messageHandler = new MessageHandler(this.consoleService);
-          MessageHandlerRegistry.add(fqn, messageHandler);
-        }
-        let lintHandler = LintHandlerRegistry.get(this.appRoot);
-        if (!lintHandler) {
-          lintHandler = new LintHandler(this.linterService, this.appRoot, this.apiVersion);
-          LintHandlerRegistry.add(this.appRoot, lintHandler);
-        }
-
-        const newBuildAction = newBuild(
-          {
-            appRoot: this.appRoot,
-            toolkitRootPath,
-            fqn,
-            postBuildAction: this.action
+    if (this.checkAtomIdeUi() === true) {
```

Comment:
```suggestion
    if (this.checkAtomIdeUiInstalled()) {
```

Reply:
I think === true is necessary here since if atom-ide-ui is not installed the function returns the notification, it doesn't return false.

Reply:
but I can use the atomIdeUiPackageInstalled general variable instead